ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/DeliveryDestinationsForm.tsx

```diff
@@ -38,8 +47,39 @@ function DeliveryDestinationsForm({ title, formik }: DeliveryDestinationsFormPar
     const hasNotifierWriteAccess = hasReadWriteAccess('Integration');
     const { keyFor } = useIndexKey();
 
+    const {
+        isModalOpen: isEmailTemplateModalOpen,
+        openModal: openEmailTemplateModal,
+        closeModal: closeEmailTemplateModal,
+    } = useModal();
```

Comment:
Sure, we can use that instead

Reply:
I don't think it needs to be switched here, just wanted to point it out since I wasn't aware of `useModal()`. More of a mental note that we have two hooks that can be combined/refactored in the future when we have bandwidth to get to it.